### DISCUSSION TITLE
[Feature] Transcoded video details (size, bitrate, size compared to original)

### DISCUSSION THREAD

**Discussion by Cakeday** [immich-app/immich#9383] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be nice to see further stats in the information section for individual assets like:
1. Size of the transcoded video
2. Bitrate
3. Size as % or original

These three things are derivable from info in the HTTP request already (i.e. Range: bytes=0-20822087), with no backend support needed. These would help users tune their transcoding settings well + let users know if a particular video _has_ been transcoded already without having to check the db. I think this would also be a good first issue.

An expansion on this would involve storing the transcoding settings applied to a particular transcoded video so that users could view this, but this would require backend support.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by NicholasFlamy** [comment]
Are you aware that the transcoded does not replace the original? It's simply used for streaming to the web or mobile app, primarily if you have a weak device, a bad internet connection, or you want to save mobile data. 

Just confirming this. Even if you want it knowing the above information I understand your point but I don't think it's a priority. Maybe in the upcoming Utitilites page there could be a page which shows the transcoded videos compared to the originals.

**Reply by RWDai** [reply]
This requirement seems to be a super weak requirement

**Comment by pyorot** [comment]
yep, hard agree that info pane should show transcode info. more than that, codec info as well

however, this could lead to bugs due to the lack of cache invalidation (#6955) where the info pane details desync from the asset being previewed, and the user gets confused as to why a supported file isn't showing up

**Comment by salvo-glitch** [comment]
Knowing the resulting file size of trascoding could be useful for fine-tuning the transcoding parameters.

**Comment by himankpathak** [comment]
Just having a flag in the Info section indicating whether this video file has been transcoded or not will also be very helpful!
